.github/workflows/nightly-release.yaml

```diff
@@ -0,0 +1,43 @@
+name: Nightly Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 2 * * *" # release nightly at 2AM
+
+permissions:
+  contents: read
+
+env:
+  # renovate: datasource=github-tags depName=goreleaser-pro packageName=goreleaser/goreleaser-pro
+  GORELEASER_PRO_VERSION: v2.7.0
```

Comment:
```suggestion
  GORELEASER_PRO_VERSION: v2.11.1
```

Reply:
Can we make this change separately?

Reply:
You mean the uprade, @mx-psi? Or the whole file? The version I suggested matches all the other files. It would be weird to have the nightlies running on an older version.